guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -117,6 +117,12 @@ ifdef::katello[]
 # dnf module disable pulpcore
 ----
 endif::[]
+. Switch to the PostgreSQL 13 module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf -y module switch-to postgresql:13
+----
```

Comment:
The same step will be needed for `common/modules/proc_upgrading-a-disconnected-project-server.adoc` as well?

Reply:
No, disconnected is only documented for Satellite, and there we use a different approach.